server/container_create.go

```diff
@@ -393,6 +393,11 @@ func (s *Server) getAppArmorProfileName(annotations map[string]string, ctrName s
 	}
 
 	if profile == apparmor.ProfileRuntimeDefault {
```

Comment:
If the user is using the default profile and hasn't set up their own, then the default value of `config.ApparmorProfile` is `ocid-default`. By default, the value of `config.ApparmorProfile` is `ocid-default`. Maybe you should update your config file `/etc/ocid/ocid.conf`.